summit.html

```diff
@@ -50,43 +50,383 @@ <h1 class="section-heading">The practicalities</h1>
+              <td>9:00</td>
+            </tr>
+
+            <tr>
+              <td>
+                <details>
+                  <summary>Presentation of Tietoevry</summary>
+                  <div>
+                  </div>
+                </details>
+              </td>
+              <td><span class="tag bd-success">Beginner</span></td>
+              <td>9:30</td>
+            </tr>
+
+            <tr><td><i>Break</i></td><td></td><td>10:00</td></tr>
+
+            <tr>
+              <td>
+                <details>
+                  <summary>Log analysis and visualization using AI</summary>
+                  <div>
+                    <p>
+                      <b>Speaker:</b>Liubov Guseva
+                    </p>
+                  </div>
+                </details>
+              </td>
+              <td><span class="tag bd-success">Beginner</span></td>
+              <td>10:15</td>
+            </tr>
+
+            <tr>
+              <td>
+                <details>
+                  <summary>Upcoming protocol changes</summary>
+                  <div>
+                    <p>
+                      Walkthrough of changed that will be included in
+                      the next edition of Eiffel, as well as some
+                      proposed changes that haven't been merged yet.
+                    </p>
+                    <p>
+                      <b>Speakers:</b> Magnus Bäck
+                    </p>
+                  </div>
+                </details>
+              </td>
+              <td><span class="tag bd-success" style="border: 1px solid blue !important;">Intermediate</span></td>
+              <td>11:15</td>
+            </tr>
+
+            <tr><td><i>Lunch</i></td><td></td><td>12:00</td></tr>
+
+            <tr>
+              <td>
+                <details>
+                  <summary>SBOM</summary>
+                  <div>
+                    <p>
+                      <b>Speaker:</b> Helena Slívová (Tietoevry)
+                    </p>
+                  </div>
+                </details>
+              </td>
+              <td><span class="tag bd-success">Beginner</span></td>
+              <td>13:00</td>
+            </tr>
+
+            <tr>
+              <td>
+                <details>
+                  <summary>Presentation from Volvo Trucks</summary>
+                  <div>
+                    <p>
+                      <b>Speakers:</b> TBD
+                    </p>
+                  </div>
+                </details>
+              </td>
+              <td><span class="tag bd-success" style="border: 1px solid blue !important;">Intermediate</span></td>
+              <td>14:00</td>
+            </tr>
+
+            <tr><td><i>Break</i></td><td></td><td>15:00</td></tr>
+
+            <tr>
+              <td>
+                <details>
+                  <summary>Eiffel Future: Eiffel vs OTel</summary>
+                  <div>
+                    <p>
```

Comment:
```suggestion
                    OTEL (Open Telemetry) has risen as the de facto standard in observability. Recently it has expanded into CI/CD space. Let us dive into the subject and explore how this might affect Eiffel.
                    <p>
```